### PR TITLE
Add memory.used_percent for RedisCollector

### PR DESCRIPTION
**Pull request by shortdudey123**

### CODE REVIEW

File: src/collectors/redisstat/redisstat.py [hunk 1]
```diff
@@ -265,6 +265,24 @@ def _get_info(self, host, port, unix_socket, auth):
         del client
         return info
 
+    def _get_config(self, host, port, unix_socket, auth, config_key):
+        """Return config string from specified Redis instance and config key
+
+:param str host: redis host
```
Comment (jaingaurav) [inline src/collectors/redisstat/redisstat.py]: Could you please fix the indentation here?

File: src/collectors/redisstat/redisstat.py [hunk 2]
```diff
@@ -265,6 +265,24 @@ def _get_info(self, host, port, unix_socket, auth):
...
+:rtype: str
+
+        """
+
+        client = self._client(host, port, unix_socket, auth)
+        if client is None:
+            return None
+
+        config_value = client.config_get(config_key)
+        del client
```
Comment (jaingaurav) [inline src/collectors/redisstat/redisstat.py]: Why do you need to call delete?

**Comment by jaingaurav** [comment]
Looks great. If you could please address the minor comments (please ensure it is in a single squashed diff) and I'll get it merged in.

File: src/collectors/redisstat/redisstat.py (hunk 1 above)
Comment (shortdudey123) [inline src/collectors/redisstat/redisstat.py]: This keeps consistency with the `_get_info` method above, but i agree w/ you

File: src/collectors/redisstat/redisstat.py (hunk 2 above)
Comment (shortdudey123) [inline src/collectors/redisstat/redisstat.py]: Same as previous

**Comment by shortdudey123** [comment]
@jaingaurav I can change the rest of the file to be consistent with the feedback (I just have it that way for consistency) or only my new part.  Let me know either way

**Comment by jaingaurav** [comment]
That's fine, I'll pull it in as is thanks!

**Comment by shortdudey123** [comment]
No problem! thanks :)

**Pull request merged**